nss-day-cohort-41/tabloidcli-dr-teeth-the-electric-mayhem
# PR for BLOG CRUD and MENU HEADER

# BLOGZ SECTION/MENU HEADER Pull Request (x5 TICKETS):

->MAIN MENU HEADER has been added with (welcome message). 
..TICKET: MAIN MENU HEADER
->CRUD has been achieved for Blogz SECTION:
..TICKET: LIST BLOGZ
..TICKET: ADD BLOGZ
..TICKET: REMOVE BLOGZ
..TICKET: EDIT BLOGZ

No Dependencies, the changes to the blog section should not affect any other section….as tested with Authors section. Currently all tests come back clear and without error in log.

Fixes # (issue)
## Type of change
- [ ] Added Files: BlogManager.cs, BlogRepository.cs, and changes too MainMenuManager.cs
- [ ] New feature: Added ASCii files to main menu (welcome/goodbye). Refactored my code, commented out my sections, and slowly tested in sections. My Blog section closely resembles that of the already complete authors section. You can look at the full list of blog posts without error & select the post you would like to edit/remove. This process has been tested and verified without issue/bug.  As well, you can create and add a blog post and then call on the list option with a verified post displaying!
 
# Testing Instructions
Upon running: dotnet run…
-Press any key.. (Menu Header will appear)
-Press any Key.. (Menu will display)
-Press 2 and 1..(List of blogz will appear)
-Press 3 and follow the input dialogue..(A Blog will be added, Press 2/1 to see new list)
-Press 4 and select blog post and follow input dialogue..(you will be able to edit a post)
-Press 5 and select blog post..(you can delete a blog post)


# Checklist:
- [ yes ] My code follows the style guidelines of this project
- [ yes ] I have performed a self-review of my own code
- [ yes ] I have commented my code, particularly in hard-to-understand areas
- [ yes ] My changes generate no new warnings or errors
- [ yes ] I have added test instructions that prove my fix is effective or that my feature works
